auth/lib/models/code_display.dart

```diff
@@ -12,6 +13,9 @@ class CodeDisplay {
   String note;
   final List<String> tags;
   int position;
+  String customIconData;
+  bool isCustomIcon;
+  IconType iconType;
```

Comment:
Instead of defining type, let's just use src
String iconSrc;
String iconID;

isCustomIcon can just be a getter => iconSrc != '' && iconID |=''l